src/pybind/mgr/dashboard/services/rbd.py

```diff
@@ -101,6 +101,14 @@ def _list(ioctx):
                 with rbd.Image(ioctx, self._image_name) as image:
                     result = image.config_list()
             else:  # pool config
+                pg_status = CephService.get_pool_pg_status(self._pool_name)
+                if 'creating+incomplete' in pg_status:
```

Comment:
1. If the issue is with the `incomplete` status (is it?), I'd filter out by that one alone.

1. Nonetheless, I see librados offers [`get_inconsistent_pgs`](https://docs.ceph.com/docs/master/rados/api/librados/#c.rados_inconsistent_pg_list). I wonder if `incosistent` too blocks on librbd. Perhaps we could have more inputs from @dillaman, @liewegas (or anyone else with deep RADOS expertise) on which statuses will block indefinitely...?

1. Additionally, as soon as Dashboard is used more often on the field we might encounter more PG statuses that result in blocking/underperforming calls. Wouldn't it make sense to expose that list as a dashboard OPTION (e.g.: `blocking_pg_statuses="incomplete,inconsistent,..."`), so it can be easily extended (or removed) with no need of code changes?

Reply:
> Practically every RADOS command will fail if you point it at a bad pool. It's not going to be worth the effort to one-off these corner-cases.

https://tracker.ceph.com/issues/43771

Reply:
@epuertat now i test if the status key includes `incomplete`. It seems to me that according to [the documentation](https://docs.ceph.com/docs/mimic/rados/operations/pg-states/) `incomplete` is the worst state as it requires bigger actions to fix it.